packages/mls-client/rollup.config.js

```diff
@@ -0,0 +1,59 @@
+import json from '@rollup/plugin-json'
+import typescript from '@rollup/plugin-typescript'
+import { defineConfig } from 'rollup'
+import { dts } from 'rollup-plugin-dts'
+import filesize from 'rollup-plugin-filesize'
+import tsConfigPaths from 'rollup-plugin-tsconfig-paths'
+
+const external = [
+  'node:path',
+  'node:process',
+  '@xmtp/mls-client-bindings-node',
+  '@xmtp/proto',
+  '@xmtp/xmtp-js',
+]
+
+const plugins = [
+  tsConfigPaths(),
+  typescript({
+    declaration: false,
+    declarationMap: false,
+  }),
+  filesize({
+    showMinifiedSize: false,
+  }),
+  json({
+    preferConst: true,
+  }),
+]
+
+export default defineConfig([
```

Comment:
this declaration is very cool. You think we can do something like this inside the bot package for declaring ts and build scripts